README.md

```diff
@@ -13,6 +13,12 @@ The Ramani Blog is a visionary project designed to shine a spotlight on the rema
     $ npm install
     $ npm run dev
 
+## Resources
```

Comment:
@Jkarage  Thank you for adding educational resources! Could you please add a bit more resources for each tech stack item? Perhaps include some videos and articles. Try to cater to different learners, visual learners. Also since this is a blog, perhaps even an article on how to build a blog using Next, etc.

Reply:
@lytes20 Awesome, I will add more resources as suggested.